src/thriveopedia/fossilisation/FossilisedSpecies.cs

```diff
@@ -0,0 +1,243 @@
+using Path = System.IO.Path;
+
+/// <summary>
+///    A species saved by the user. Contains helper methods for saving/loading species on the disk.
+/// </summary>
+public class FossilisedSpecies
+{
+    public const string SAVE_SAVE_JSON = "save.json";
+    public const string SAVE_INFO_JSON = "info.json";
+
+    /// <summary>
+    ///   Name of this saved species on disk.
+    /// </summary>
+    public string Name { get; set; } = "invalid";
+
+    /// <summary>
+    ///   General information about this saved species.
+    /// </summary>
+    public FossilisedSpeciesInformation Info { get; set; } = new();
+
+    /// <summary>
+    ///   The species to be saved/loaded.
+    /// </summary>
+    public Species Species { get; set; } = null!;
+
+    /// <summary>
+    ///   Creates a list of existing fossilised species names to prevent unintended overwrites.
+    /// </summary>
+    /// <returns>A list of names of .thrivefossil files in the user's home directory</returns>
+    public static List<string> CreateListOfSaves()
+    {
+        var result = new List<string>();
+
+        using (var directory = new Directory())
+        {
+            if (!directory.DirExists(Constants.FOSSILISED_SPECIES_FOLDER))
+                return result;
+
+            directory.Open(Constants.FOSSILISED_SPECIES_FOLDER);
+            directory.ListDirBegin(true, true);
+
+            while (true)
+            {
+                var filename = directory.GetNext();
+
+                if (string.IsNullOrEmpty(filename))
+                    break;
+
+                if (!filename.EndsWith(Constants.FOSSIL_EXTENSION, StringComparison.Ordinal))
+                    continue;
+
+                // Skip folders
+                if (!directory.FileExists(filename))
+                    continue;
+
+                result.Add(filename);
+            }
+
+            directory.ListDirEnd();
+        }
+
+        using var file = new File();
+        result = result.OrderBy(item =>
+            file.GetModifiedTime(Path.Combine(Constants.FOSSILISED_SPECIES_FOLDER, item))).ToList();
+
+        return result;
+    }
+
+    /// <summary>
+    ///   Loads a fossilised species by its filename.
+    /// </summary>
+    /// <param name="saveName">The name of the .thrivefossil file (including extension)</param>
+    /// <returns>The species saved in the provided file</returns>
+    public static Species LoadSpeciesFromFile(string saveName)
+    {
+        var target = Path.Combine(Constants.FOSSILISED_SPECIES_FOLDER, saveName);
+        var (_, species) = LoadFromFile(target);
+
+        return species;
+    }
+
+    /// <summary>
+    ///   Saves this species to disk.
+    /// </summary>
+    public void SaveToFile()
+    {
+        if (Species is not MicrobeSpecies)
+        {
+            throw new NotImplementedException("Saving non-microbe species is not yet implemented");
+        }
+
+        // For now, save everything as a microbe to aid deserialization
```

Comment:
Well actually due to the dynamic species saving already, it should be possible to really do without the Type etc. fuss here, but unless tested, I think having the NotImplementedException here is still a good idea.